.github/workflows/flake8.yml

```diff
@@ -21,7 +21,10 @@ jobs:
     - name: Install pre-commit
       run: |
         python -m pip install --upgrade pip wheel
-        pip install -r requirements-dev.txt
+        pip install ruff -r requirements-dev.txt
```

Comment:
Please avoid installing ruff manually, and especially without a version pin.

Reply:
Done.